docs/en/latest/deployments/aws.md

```diff
@@ -38,7 +38,9 @@ This document explains how to install Ingress APISIX on [Amazon EKS](https://ama
 ```shell
 cd /path/to/apisix-helm-chart
 helm repo add bitnami https://charts.bitnami.com/bitnami
-helm dependency update ./charts/apisix
+helm repo add apisix https://charts.apiseven.com
+# Use `hem search repo apisix` to search charts about apisix
+helm repo update
 helm install apisix ./charts/apisix \
```

Comment:
We don't need to use local chart as we use the online repo already.

Reply:
Update the commit.